model/project_parser.go

```diff
@@ -567,6 +567,11 @@ func LoadProjectInto(ctx context.Context, data []byte, opts GetProjectOpts, iden
 	// return intermediateProject even if we run into issues to show merge progress
 	for _, path := range intermediateProject.Include {
 		opts.RemotePath = path.FileName
+		if opts.PatchOpts.patch != nil {
```

Comment:
can combine these if statements with &&

Reply:
Should also add a comment to explain why we're doing this. (And in the else case, we should change it to ReadFromPatch, so that with multiple includes we switch back if necessary)

Reply:
i think we would want to limit that behavior only if it's reading from a patch